src/aggregate_poisson.py

```diff
@@ -0,0 +1,339 @@
+DATA_STORE = DATA_STORE = '/home/ps35324/gpu-sharing-scheduler/src/data-store/scheduler.db'
```

Comment:
This is an issue. Please avoid using hard coded paths. Can this be given as an input parameter?